larky/src/main/java/com/verygood/security/larky/modules/vgs/cerebro/text/analyzer/analyzers/DefaultTextPIIAnalyzer.java

```diff
@@ -0,0 +1,56 @@
+package com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.analyzers;
+
+import com.google.common.collect.Sets;
+import com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.dto.TextPIIEntity;
+import com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.spi.TextPIIAnalyzer;
+import net.starlark.java.eval.StarlarkFloat;
+import net.starlark.java.eval.StarlarkInt;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DefaultTextPIIAnalyzer implements TextPIIAnalyzer {
+
+  private static final Set<String> SUPPORTED_LANGUAGES = Sets.newHashSet("EN");
+
+  private static final Set<String> SUPPORTED_PII_ENTITIES = Sets.newHashSet("CREDIT_CARD");
+
+  private static final Pattern CREDIT_CARD_PATTERN = Pattern.compile("\\b(?:\\d[ -]*?){13,16}\\b");
```

Comment:
Would we want the configurations to be passed in Via larky as well or do we want this to be hard coded and maintained by us?

Reply:
My idea is that default one should be simple and just demonstrate the basic ideas, encouraging to build implementations or using platforms with more advanced implementations.